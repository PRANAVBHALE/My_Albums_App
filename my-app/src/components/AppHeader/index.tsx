import React from 'react';
import { Layout } from 'antd';

const { Header } = Layout;

const headerStyle: React.CSSProperties = {
  // textAlign: 'center',
  color: '#36454F',
  height: 64,
  paddingInline: 50,
  lineHeight: '64px',
  backgroundColor: '#02ec88',
  fontSize: '24px',
  fontWeight: "600"
};

const AppHeader = () =>  <Header style={headerStyle}>Albums Application</Header>

export default AppHeader