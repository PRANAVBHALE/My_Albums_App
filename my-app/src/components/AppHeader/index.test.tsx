import { screen, render } from "@testing-library/react";
import AppHeader from ".";

describe("AppHeader component", () => {
  it("should render and match snapshot", () => {
    let view = render(<AppHeader />);

    expect(view).toMatchSnapshot();
  });

  it('Albums Application"', () => {
    render(<AppHeader />);
    let query = screen.getByText("Albums Application");

    expect(query).toBeInTheDocument();
  });
});
