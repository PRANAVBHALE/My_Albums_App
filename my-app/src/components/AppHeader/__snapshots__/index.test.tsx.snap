// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader component should render and match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="ant-layout-header css-dev-only-do-not-override-15rg2km"
        style="color: rgb(54, 69, 79); height: 64px; padding-inline: 50px; line-height: 64px; background-color: rgb(2, 236, 136); font-size: 2vw; font-weight: 600;"
      >
        Albums Application
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="ant-layout-header css-dev-only-do-not-override-15rg2km"
      style="color: rgb(54, 69, 79); height: 64px; padding-inline: 50px; line-height: 64px; background-color: rgb(2, 236, 136); font-size: 2vw; font-weight: 600;"
    >
      Albums Application
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
